.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -7,22 +7,41 @@ assignees: ""
 ---
 
 <!--
-Please see https://github.com/cdr/code-server/blob/master/docs/FAQ.md#how-do-i-debug-issues-with-code-server
-and include any logging information relevant to the issue.
 
-Please search for existing issues before filing.
+Hi there! 👋
 
-If you can reproduce the issue on vanilla VS Code,
-please file the issue at the VS Code repository instead.
+Thanks for reporting a bug. Please see https://github.com/cdr/code-server/blob/main/docs/FAQ.md#how-do-i-debug-issues-with-code-server and include any logging information relevant to the issue.
 
-Provide screenshots if applicable.
+Please search for existing issues before filing.
 
-Please fill in the issue template and try to be as detailed
-and clear as possible!
+Failure to fill this out may result in your issue being closed!
```

Comment:
I think the tone of what we had before was kinder than this, I would suggest rephrasing in terms of what we need and why we need it (it's understandable that bug reports need as much detail as possible, otherwise we can't reproduce problems and don't have much to go on).

Perhaps:

> Please search for existing issues before filing, as they may contain additional information about the problem and descriptions of workarounds. Provide as much information as you can, so that we can reproduce the issue. Otherwise, we may not be able to help diagnose the problem, and may close the issue as unreproducible or incomplete. For visual defects, please include screenshots to help us understand the issue.

Reply:
Good call! I was trying to come off more serious but may have lost some empathy in that line. I'll use what you suggested!

Reply:
Thanks for the quick review by the way! 🙌